ebos/equil/initstateequil.hh

```diff
@@ -128,12 +128,14 @@ private:
                    const double press) const;
 };
 
-template <class FluidSystem, class RV>
+//TODO:add argument rvw
+template <class FluidSystem, class RV, class RVW>
```

Comment:
Is this comment left over from earlier development or is there still something missing here?

Reply:
This is an obsolete comment. I removed it.